### ISSUE TITLE
list of tables

### ISSUE THREAD

**Issue by kurtbern** [brentp/cruzdb#23]
Hi Brent!
Can you tell me where I can get an overview of all the tables(like cpgIslandExt) of for example mm10?
I'm using cruzdb from the command line: python -m cruzdb mm10 metilene.bed cpgIslandExt ...

**Comment by adelq** [comment]
I don't think you can get a list of all tables through the command line, though it's straightforward enough with a bit of Python code. It may be a good idea to rethink using all tables though, as there are currently 600+ for mm10.

```python
from cruzdb import Genome
g = Genome(db="mm10")
print g.engine.table_names()
```
